### PR TITLE
working script

### PR DESCRIPTION
**Pull request by scottty881**

### CODE REVIEW

**Comment by scottty881** [comment]
note: there is no build on this repo

File: config/config.example.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+
+module.exports = {
+  sqlToRunDirectory: './completed_dmls',
+  ranSqlDirectory: './dmls_to_run',
```
Comment (maxsmithrx) [inline config/config.example.js]: I think these two should be swapped?

File: package.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
+{
+  "name": "@smithrx/user-portal-dmls",
+  "version": "0.0.2",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "start": "node index.js"
```
Comment (maxsmithrx) [inline package.json]: I may have been doing it wrong, but I couldn't get "npm run start" to work yesterday.  It said I needed to pass an env parameter as --env=ENVIRONMENT, but "npm run start --env=test" and "npm run --env=test start" didn't work.  I had to call "node index.js --env=test" directly

Comment (scottty881) [inline package.json]: try `npm run start -- --env= ENVIRONMENT`

Comment (maxsmithrx) [inline package.json]: Knew it was probably user error, that seems to work the same as the node index.js version

**Pull request merged**